servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -3055,6 +3499,30 @@
     }
   },
   "definitions": {
+    "RegisterClusterPackage": {
+      "type": "object",
+      "description": "The package of the register cluster",
+      "properties": {
+        "CodeFilePath": {
+          "type": "string"
+        },
+        "ClusterManifestFilePath": {
+          "type": "string"
```

Comment:
do you accept relative or absolute file paths or both? Can you please indicate that in the description?